src/themes/default/components/core/blocks/Checkout/OrderReview.vue

```diff
@@ -89,18 +96,14 @@ export default {
 
   .checkboxStyled {
     width: 23px;
-    position: relative;
     display: inline-block;
```

Comment:
we should add `inline-block` in `positioning.scss` (and maybe change the name to `layout.scss`?

Reply:
Do we need `inline block`? If out layout is based on flex perhaps it would be better to avoid this property.

Reply:
RIght, I'll convert `inline-block` to flex, we have only few instances of this property in the project.

Reply:
Done